Achievements/SelfMade.lua

```diff
@@ -57,15 +57,14 @@ local function isSelfCreated(...)
                     return true
                 end
             end
```

Comment:
Thats odd, why were we using 18?

Reply:
because i had set it really high and it would never fail and then i had it set low and it would never pass, so i found 18 worked for all the gear I had at the time and when i counted the nils and the actual lines that seemed to be the amount that triggered it ....

Reply:
I'm confused though. The for loop is already cycling through to the max number of regions right?

Reply:
Um yup, you were right, just me being dumb and not putting the false check outside the for loop....